OpenSim/Simulation/Model/Smith2016Ligament.h

```diff
@@ -0,0 +1,277 @@
+#ifndef _OPENSIM_SMITH_2016_LIGAMENT_H_
+#define _OPENSIM_SMITH_2016_LIGAMENT_H_
+/* -------------------------------------------------------------------------- *
+ *                     OpenSim:  Smith2016Ligament.h                          *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2019 Stanford University and the Authors                *
+ * Author(s): Colin Smith                                                     *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+#include <OpenSim/Simulation/Model/Force.h>
+#include <OpenSim/Simulation/Model/GeometryPath.h>
+
+
+
+namespace OpenSim {
+
+//==============================================================================
+//                         Smith2016LigamentModel
+//==============================================================================
+/**
+This class implements a nonlinear spring ligament model as described in
+Smith et al.\ (2016). This model is based on the formulation orginally 
+proposed by Wismans et al.\ (1980) and Blankevoort et al.\ (1991). The ligament
+is represented as a nonlinear spring with a quadratic "toe" region and a linear
+region. Additionally, the ligament includes a normalized damping force.   
+\image html fig_Smith2016Ligament.png
+<B>Governing Equations</B>
+Spring Force:
+
+Damping Force:
+
+Total Force:
+
+An equilibrium model assumes that the forces generated
+by the fiber and tendon are equal:
+\f[
+ f_{ISO}\Big(\mathbf{a}(t) \mathbf{f}_L(\hat{l}_{CE}) \mathbf{f}_V(\hat{v}_{CE})
```

Comment:
Consider wrapping subscripts that have multiple letters in `\textrm{}` so they are not italicized.